phpcs.xml.dist

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<ruleset name="WP-CLI-rewrite">
+	<description>Custom ruleset for WP-CLI rewrite-command</description>
+
+	<!--
+	#############################################################################
+	COMMAND LINE ARGUMENTS
+	For help understanding this file: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml
+	For help using PHPCS: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Usage
+	#############################################################################
+	-->
+
+	<!-- What to scan. -->
+	<file>.</file>
+
+	<!-- Ignoring select files/folders.
```

Comment:
@pattonwebz We would to remove this whole section, this has been done in other repos too.
Ref: https://github.com/wp-cli/maintenance-mode-command/pull/3/commits/6187b8aaae88b1de81895499c67e36032a04b939